src/main/kotlin/net/irisshaders/lilybot/extensions/events/MessageDelete.kt

```diff
@@ -10,27 +10,35 @@ import dev.kord.core.event.message.MessageDeleteEvent
 import kotlinx.datetime.Clock
 import net.irisshaders.lilybot.utils.DatabaseHelper
 
+/**
+ * The class for logging deletion of messages to the guild action log.
+ *
+ * @since 2.0
+ */
 class MessageDelete : Extension() {
 	override val name = "message-delete"
 
 	override suspend fun setup() {
 		/**
 		 * Logs deleted messages in a guild to the message log channel designated in the config for that guild
 		 * @author NoComment1105
+		 * @since 2.0
 		 */
 		event<MessageDeleteEvent> {
 			// Don't try to create if the message is in DMs
```

Comment:
This comment really isn't needed, it's blindingly obvious and doesn't exist anywhere else